acpi.adoc

```diff
@@ -1,26 +1,142 @@
 [[acpi]]
 == ACPI Requirements
-=== ACPI Version
-=== ACPI-provided Data Structures
-=== ACPI Tables
-==== Mandatory ACPI Tables
-===== RSDP
-===== XSDT
-===== FADT
-===== DSDT and SSDT
-===== MADT
-===== RHCT
-===== SPCR
+
+The Advanced Configuration and Power Interface specification provides the OS-centric view of system configuration, various hardware resources, events and power management.
+
+This section defines mandatory and optional ACPI requirements on top of the ACPI specification cite:[ACPI]. Additional non-normative guidance may be found in <<ACPI Implementation Guidance>>.
+
+=== ACPI High-Level Requirements
+
+A compliant platform must:
+
+* Conform to UEFI specification requirements for ACPI:
+** RISC-V Platforms (cite:[UEFI] Section 2.3.7)
+** Industry Standard Configuration Tables (cite:[UEFI] Section 4.6.1.1)
+* Conform to version 6.6 of the ACPI specification, including RISC-V requirements listed in:
+** Multiple APIC Description Table (MADT, cite:[ACPI] Section 5.2.12)
+** RISC-V Hart Capabilities Table (RHCT, cite:[ACPI] Section 5.2.36)
+* Meet the requirements in <<Minimum Hardware Requirements Imposed By ACPI>>
+* Be 64-bits clean.
+** RSDT must not be implemented.
+** 32-bit address fields must be 0.
+* Implement a firmware-first APEI model (cite:[ACPI] Section 18.4)
+
+=== ACPI-specific Hardware Requirements
+
+This section lists the BRS-specific hardware requirements imposed by
+ACPI support.
+
+==== HW-Reduced ACPI Model
+
+Compliant RISC-V platforms only implement the HW-Reduced ACPI Model. This means the hardware portion of
+cite:[ACPI] (Section 4) is not required or supported. All functionality is instead provided through equivalent
+software-defined interfaces and the complexity in supporting ACPI is reduced.
+
+The following additional BRS requirements apply:
+
+* Both interrupt-signaled and GPIO-signaled ACPI Events are supported
+  (cite:[ACPI] Section 5.6)
+* There is at least 1 wake event.
+** Note: for systems that do not support Sx states except S5 soft off, this can be just the power button.
+* For the ACPI Platform Error Interfaces (APEI) support:
+** There is 1 event for non-fatal error signaling (cite:[ACPI] Section 18.3.2.7.2)
+** There is 1 NMI-equivalent signal for use in fatal errors (cite:[ACPI] Section 18)
```

Comment:
Do we not need to go into the definition of event vs signal with these two items?

Reply:
To the extent these are used/defined by the ACPI spec - I think the answer is no. We appropriate reference the ACPI APEI content and understanding that is a prerequisite.